git_machete/generated_long_docs.py

```diff
@@ -0,0 +1,960 @@
+           <b>-C</b>, <b>--checked-out-since=<date></b>
+              Only consider branches checked out at least once since the given date. <date> can be e.g. `2 weeks ago` or `2020-06-01`, as in `git log --since=<date>`. If not present, the date is selected automatically so that around 10 branches are included.
+           <b>-l</b>, <b>--list-commits</b>
+              When printing the discovered tree, additionally lists the messages of commits introduced on each branch (as for `git machete status`).
+           <b>-r</b>, <b>--roots=<branch1,...></b>
+              Comma-separated list of branches that should be considered roots of trees of branch dependencies. If not present, `master` is assumed to be a root. Note that in the process of discovery, certain other branches can also be additionally deemed to be roots as well.
+           <b>-y</b>, <b>--yes</b>
+              Don't ask for confirmation before saving the newly-discovered tree. Mostly useful in scripts; not recommended for manual use.
+   """,
+    "edit": """
+        <b>Usage:</b><b>
+           git machete e[dit]</b>
+
+        Opens an editor and lets you edit the definition file manually.
+
+        The editor is determined by checking up the following locations:
+           * `$GIT_MACHETE_EDITOR`
+           * `$GIT_EDITOR`
+           * `$(git config core.editor)`
+           * `$VISUAL`
+           * `$EDITOR`
+           * `editor`
+           * `nano`
+           * `vi`
+
+        and selecting the first one that is defined and points to an executable file accessible on `PATH`.
+
+        Note that the above editor selection only applies for editing the definition file,
+        but not for any other actions that may be indirectly triggered by git machete, including editing of rebase TODO list, commit messages etc.
+
+        The definition file can be always accessed and edited directly under the path returned by `git machete file`
+        (usually `.git/machete`, unless worktrees or submodules are involved).
+
+        <b>Environment variables:</b>
+           `GIT_MACHETE_EDITOR`
+              Name of the editor executable.
+
+   """,
+    "file": """
+        <b>Usage:</b><b>
+           git machete file</b>
+
+        Outputs the absolute path of machete definition file.
+        The file is always called `machete` and is located in the git directory of the project.
+
+        Three cases are possible:
+           * if `git machete` is executed from a regular working directory (not a worktree or submodule), the file is located under `.git/machete`,
+           * if `git machete` is executed from a <b>worktree</b>, the file path depends on the `machete.worktree.useTopLevelMacheteFile` config key value:
+              - if `machete.worktree.useTopLevelMacheteFile` is true (default), the file is located under `.git/machete`
+              - if `machete.worktree.useTopLevelMacheteFile` is false, the file is located under `.git/worktrees/.../machete`,
+
```

Comment:
Nit: any chance to remove this newline, or to add a newline b/w 1st and 2nd top-level bullets for consistency?

Reply:
@PawelLipski Unfortunately it's not easily doable, i will look into it next week.

Reply:
Okay, pls open a separate issue for that so as not to block this PR